internal/driver/glfw/menu.go

```diff
@@ -5,7 +5,15 @@ import (
 )
 
 func buildMenuOverlay(menus *fyne.MainMenu, c fyne.Canvas) fyne.CanvasObject {
-	if menus.Items[0].Items[len(menus.Items[0].Items)-1].Label != "Quit" { // make sure the first menu always has a quit option
+	if len(menus.Items) == 0 {
+		fyne.LogError("Main menu must have at least one child menu", nil)
```

Comment:
I don't think this should be an error since an empty menu will get the default "Quit" option.